tests/integration/local/start_api/test_start_api.py

```diff
@@ -148,6 +148,39 @@ def test_large_input_request_http10(self):
         self.assertEqual(response.raw.version, 11)
 
 
+class TestParallelRequestsZipped(StartApiIntegBaseClass):
```

Comment:
If this is the same test as the one below (`TestParallelRequests`), we could parallelize the endpoint instead.